lib-api/src/main/java/org/ergoplatform/appkit/InputBox.java

```diff
@@ -32,5 +33,7 @@
     Values.ErgoTree getErgoTree();
 
     String toJson(boolean prettyPrint);
+
+    Optional<ErgoId> getSpentTransactionId();
```

Comment:
What this method should do? What does it mean "spent transaction"? Transaction cannot be spent.

Reply:
It's an id of the transaction where the box was used as input. See http://github.com/ergoplatform/explorer-back/blob/73096d2e630609bcee0de1b4cb42ad8d81e445d1/src/main/resources/api/openapi.yaml#L866-L866
But seems like it does not belong to `InputBox` at all. We need a type to represent `TransactionOutput` with a more generic class. Maybe `TxOutBox`?